packages/engine-test-utils/src/__tests__/engine-server/configUtils.spec.ts

```diff
@@ -9,6 +9,64 @@ function getConfig(
 }
 
 describe("ConfigUtils", () => {
+  describe("GIVEN findDifference", () => {
+    describe("WHEN config has no difference", () => {
+      test("THEN correctly outputs no change", () => {
+        const config = getConfig();
+        const output = ConfigUtils.findDifference({ config });
+        expect(output.length).toEqual(0);
+      });
+    });
+    describe("WHEN v4 config is given", () => {
+      test("THEN return empty list", () => {
+        const config = ConfigUtils.genDefaultV4Config();
+        const output = ConfigUtils.findDifference({ config });
+        expect(output.length).toEqual(0);
+      });
+    });
+    describe("WHEN changed config is given", () => {
```

Comment:
can we also add a test where the config only differes by vaults? we don't want to emit a diff in that case